kasa/cli/main.py

```diff
@@ -465,6 +258,9 @@ def _nop_echo(*args, **kwargs):
 
     if alias is not None and host is None:
         echo(f"Alias is given, using discovery to find host {alias}")
+
+        from kasa.cli.discover import find_host_from_alias
```

Comment:
Should probably use relative imports as both modules are inside the cli package? Applies to other imports so I don't mention them again for this file.

Reply:
Done